robot/util/emailer/syncEmailer.sh

```diff
@@ -1,3 +1,14 @@
-cp ./* /home/odroid/emailer
-cd /home/odroid/emailer/
+EMAILER_DIR="/home/odroid/emailer"
+
+if [ ! -d $EMAILER_DIR ]; then
+    mkdir $EMAILER_DIR;
+else
+    echo "There already exists an emailer directory in /home/odroid"
+    echo "Please rename or delete the folder for this script to proceed"
+    exit 1
```

Comment:
what if you updated the emailer code and want to recopy it into /home/odroid/emailer ? I don't think it should fail when the folder already exists

Reply:
so you're saying it would be best to simply overwrite anything in there?